### DISCUSSION TITLE
Integrating Stumpy into OpenBB (partnership opportunity)

### DISCUSSION THREAD

**Discussion by martinb-ai** [TDAmeritrade/stumpy#635] [Ideas]
Hi there, 

I lead ML at https://github.com/OpenBB-finance/OpenBBTerminal.

I think this repository would be perfect to integrate into our open source financial research terminal. I would love to chat more potentially offline to see how OpenBB and TD could partner on bringing this tool to the masses. 

Thank you 😄

**Comment by seanlaw** [comment]
@martinb-bb Thanks for opening this discussion and welcome to the STUMPY community. I just listened to the OpenBBTerminal episode on TalkPython. Happy to talk about what you might have in mind! Let's set up some time.
